File: problems/t/triarea/triarea.c
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <stdio.h>

int main(void) {
    int a, b;
    float triangle_area;
    if (scanf("%d%d", &a, &b) != 2) {
        printf("Failed to read a and b!");
    }

    triangle_area = (a * b) / 2.0;

    printf("%f", triangle_area);

    return 0;
}

File: problems/t/triarea/triarea.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    a, b = [int(x) for x in input().split()]
    triangle_area = (a * b) / 2
    print(triangle_area)

if __name__ == '__main__':
    main()